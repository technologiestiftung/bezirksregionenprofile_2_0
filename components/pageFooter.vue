<template>
  <div class="row">
    <div class="span12">
      <!-- 4 template-path: tmpl_default/footer/footer_content.php -->
      <!-- template start: footer-content -->
      <div class="html5-footer content-footer" role="contentinfo">
        <div class="html5-nav">
          <ul class="nav">
            <li><nuxt-link class="ic-fa-impressum" to="/impressum">Impressum</nuxt-link></li>
            <li><nuxt-link class="ic-fa-info" to="/about">Über das Projekt</nuxt-link></li>
            <li><nuxt-link class="ic-fa-lock" to="/datenschutz">Datenschutzerklärung</nuxt-link></li>
            <li><nuxt-link class="ic-fa-access" to="/barrierefreiheit">Digitale Barrierefreiheit</nuxt-link></li>
            <li class="right"><a class="ic-fa-totop" href="#">zum Seitenanfang</a></li>
          </ul>
        </div>
      </div>

      <!-- /template end: footer-content -->
    </div>
  </div>
</template>
