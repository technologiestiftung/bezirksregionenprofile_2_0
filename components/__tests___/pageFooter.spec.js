import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import vuetify from 'vuetify'
import pageFooter from '../pageFooter'

describe('pageFooter.vue', () => {
  let wrapper = null
  beforeEach(() => {
    Vue.use(vuetify)
    wrapper = shallowMount(pageFooter)
  })
  it('Renders vue instance', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })
  it('has a link', () => {
    expect(wrapper.contains('a')).toBe(true)
  })
})
