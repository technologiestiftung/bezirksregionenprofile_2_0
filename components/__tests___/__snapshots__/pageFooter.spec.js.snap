// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders properly 1`] = `
"<v-footer app=\\"\\" class=\\"grey lighten-2\\"><v-container fluid=\\"\\" class=\\"px-3 pb-8 px-sm-5 px-md-8 px-lg-12\\"><v-row no-gutters=\\"\\"><v-col><ul class=\\"d-flex justify-center justify-lg-end social-icons\\"><li><a target=\\"_blank\\" href=\\"https://twitter.com/tsbberlin\\"><v-icon>mdi-twitter</v-icon></a></li> <li><a target=\\"_blank\\" href=\\"https://www.facebook.com/Technologiestiftung/\\"><v-icon>mdi-facebook-box</v-icon></a></li> <li><a target=\\"_blank\\" href=\\"https://github.com/technologiestiftung/\\"><v-icon>mdi-github-circle</v-icon></a></li></ul></v-col></v-row> <v-row no-gutters=\\"\\"><v-col><div class=\\"d-flex flex-column flex-lg-row\\"><div class=\\"d-flex flex-column flex-lg-row\\"><img alt=\\"Technologiestiftung Berlin Logo\\" src=\\"/img/tsb-logo-content.svg\\" class=\\"mr-6\\"> <address class=\\"mt-4 mt-lg-0 ml-2\\">
              Grunewaldstraße 61-62<br>
              10825 Berlin<br> <a href=\\"mailto:info@technologiestiftung-berlin.de\\">info@technologiestiftung‑berlin.de</a><br></address></div> <ul class=\\"pl-2 mt-7 mt-lg-0 ml-lg-12\\"><li><a target=\\"_blank\\" href=\\"https://www.technologiestiftung-berlin.de/de/stiftung/kontakt-anfahrt/\\">Kontakt</a></li> <li><a target=\\"_blank\\" href=\\"https://www.technologiestiftung-berlin.de/de/impressum\\">Impressum</a></li> <li><a target=\\"_blank\\" href=\\"https://www.technologiestiftung-berlin.de/de/datenschutz/\\">Datenschutz</a></li></ul></div></v-col></v-row></v-container></v-footer>"
`;
